scbw/game.py

```diff
@@ -110,6 +111,7 @@ def _wait_callback():
         map_name=args.map,
         game_type=GameType(args.game_type),
         game_speed=args.game_speed,
+        seed_override=args.seed_override,
```

Comment:
The default value is 0. Does `seed_override = 0` in the config set the seed to zero or cause no seed to be given?

Reply:
Good catch, it does in fact cause the seed to be set to zero. It turns out that BWAPI uses INT_MAX internally to indicate a non-overridden seed instead of zero as I had assumed.

I've pushed an update that will allow it to remain empty in bwapi.ini when it isn't specified on the scbw.play command line.

Thanks!